PressureSensorAdapter.java

```diff
@@ -0,0 +1,43 @@
+
+/*					Author : Francois Mukaba
+ * 					Course : Software Engineering
+ * 					Spring 2019
+ */
+import sensor.PressureSensor;
+import java.awt.Color;
+
+public class PressureSensorAdapter implements SensorAdapter {
+	private PressureSensor p;
+
+	public PressureSensorAdapter() {

```

Comment:
Adapters are supposed to take an object of the adapted class as a parameter to the constructor:
```suggestion
	public PressureSensorAdapter(PressureSensor pressureSensor) {
		this.p = pressureSensor;
	}
```